apps/extension/src/ui/domains/SendFunds/useNetworkDetails.ts

```diff
@@ -0,0 +1,32 @@
+import { Chain, CustomChain, CustomEvmNetwork, EvmNetwork } from "@talismn/chaindata-provider"
+import { useCallback, useMemo } from "react"
+import { useTranslation } from "react-i18next"
+
+import { useSendFunds } from "./useSendFunds"
+
+export const useFormatNetworkName = () => {
+  const [t] = useTranslation()
+
+  return useCallback(
```

Comment:
It's maybe a little unclear what's happening here without context, could do with a comment like `We want to add a suffix to chain names for chains with both a substrate and an evm RPC`